misk-gcp/src/main/kotlin/misk/cloud/gcp/spanner/GoogleSpannerModule.kt

```diff
@@ -34,8 +36,36 @@ class GoogleSpannerModule(
       if (config.emulator.enabled) NoCredentials.getInstance()
       else ServiceAccountCredentials.getApplicationDefault()
 
+    val retrySettings = RetrySettings.newBuilder()
+
+    config.max_attempts?.let {
+      retrySettings.setMaxAttempts(it)
+    }
+    config.total_timeout_s?.let {
+      retrySettings.setTotalTimeout(Duration.ofSeconds(it))
+    }
+    config.initial_retry_delay_ms?.let {
+      retrySettings.setInitialRetryDelay(Duration.ofMillis(it))
+    }
+    config.initial_rpc_timeout_s?.let {
+      retrySettings.setInitialRpcTimeout(Duration.ofSeconds(it))
+    }
+    config.max_retry_delay_s?.let {
+      retrySettings.setMaxRetryDelay(Duration.ofSeconds(it))
+    }
+    config.max_rpc_timeout_s?.let {
+      retrySettings.setMaxRpcTimeout(Duration.ofSeconds(it))
+    }
+    config.rpc_timeout_multipler?.let {
+      retrySettings.setRpcTimeoutMultiplier(it)
+    }
+    config.total_timeout_s?.let {
```

Comment:
this is duplicated from line 44. I think you mean to set `retry_delay_multiplier` here.

Reply:
Oof, good catch.